app/assets/javascripts/modules/get-access-logs.js

```diff
@@ -0,0 +1,73 @@
+moj.Modules.accessLogs = {
+  dropdownName: 'js-access-log-period-dropdown',
+
+  init() {
+    this.$dropdown = $(`#${this.dropdownName}`);
```

Comment:
Since this is only used once, you could have `dropdownId: '#js-access-log-...'` and `this.$dropdown = $(this.dropdownId);`

Reply:
👍